internal/pkg/renderer/renderer.go

```diff
@@ -136,10 +142,13 @@ func (r *Renderer) RenderOutput() (string, error) {
 	return buf.String(), nil
 }
 
-// prepareTemplates recurses the pack and its dependencies to populate to the
-// passed map with the templates to render along with the variables which
-// correspond.
-func prepareTemplates(p *pack.Pack, templates map[string]toRender, variables map[string]interface{}) {
+// prepareFiles recurses the pack and its dependencies to populate to the passed
+// map with the templates to render along with the variables which correspond.
+func prepareFiles(p *pack.Pack,
+	templates map[string]toRender,
+	auxFiles map[string]toRender,
+	variables map[string]interface{},
+	renderAuxFiles bool) {
```

Comment:
```suggestion
	renderAuxFiles bool,
) {
```

Just to keep it consistent with other line breaks.